src/app/page/page.component.ts

```diff
@@ -26,7 +26,7 @@ namespace Page {
       });
       this.$rootScope.$on(CLOSE_MAIN_NAV_EVENT, () => this.isNavOpen = false);
       const defaultVerticalNavState = this.$window.localStorage.getItem('defaultVerticalNavState');
-      if (defaultVerticalNavState === "show") {
+      if (defaultVerticalNavState == "show" || defaultVerticalNavState == null) {
```

Comment:
Always prefer to `===` instead of `==`.
https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons

Also this is better:
```
const defaultVerticalNavState = this.$window.localStorage.getItem('defaultVerticalNavState') || 'show';
if (defaultVerticalNavState === 'show') {
    ...
```